scripts/00_setup_fetchd_local.sh

```diff
@@ -22,7 +22,7 @@ sed -i 's/cors_allowed_origins = \[\]/cors_allowed_origins = \["\*"\]/' ~/.fetch
 
 # update the block parameters to match mainnet
 cp ~/.fetchd/config/genesis.json ~/.fetchd/config.genesis.json.bak
-jq '.consensus_params.block.max_gas = "2000000" | .consensus_params.block.max_bytes = "200000" | .consensus_params.evidence.max_bytes = "200000"' ~/.fetchd/config.genesis.json.bak > ~/.fetchd/config/genesis.json
+jq '.consensus_params.block.max_gas = "3000000" | .consensus_params.block.max_bytes = "300000" | .consensus_params.evidence.max_bytes = "300000"' ~/.fetchd/config.genesis.json.bak > ~/.fetchd/config/genesis.json
```

Comment:
Where are you getting these values from?

Reply:
you can check: 
- `fetchd query params subspace baseapp BlockParams`
- https://github.com/fetchai/FIPs/pull/5
- https://www.mintscan.io/fetchai/proposals/8

Reply:
Thank you!